apps/wing-console/console/server/src/router/app.ts

```diff
@@ -114,6 +114,34 @@ export const createAppRouter = () => {
           input?.showTests,
         );
       }),
+    "app.itemIds": createProcedure
```

Comment:
Can we name it `app.nodeIds`? They are only called items in the context of a tree view, but here they are nodes.

Reply:
yep